nixos/modules/services/search/elasticsearch.nix

```diff
@@ -82,10 +90,10 @@ in
       type = types.bool;
     };
 
-    extraConf = mkOption {
+    extraConfig = mkOption {
```

Comment:
it would be better to replace this option entirely with a new `settings` option which is of type `(pkgs.formats.yml {}).type`